app/proprietary/src/main/java/stirling/software/proprietary/security/configuration/SecurityConfiguration.java

```diff
@@ -175,10 +204,10 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                                                     .matcher("/logout"))
                                     .logoutSuccessHandler(
                                             new CustomLogoutSuccessHandler(
-                                                    applicationProperties, appConfig))
+                                                    securityProperties, appConfig, jwtService))
                                     .clearAuthentication(true)
                                     .invalidateHttpSession(true)
-                                    .deleteCookies("JSESSIONID", "remember-me"));
+                                    .deleteCookies("JSESSIONID", "remember-me", "stirling_jwt"));
```

Comment:
I dont think remember me is a thing for JWT since its stateless, lets remove? same for cookie logic around remember me